src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -272,6 +266,15 @@ internal async Task ConnectAsync(PeerAddress peerAddress)
             }
         }
 
+        /// <summary>
+        /// Determines how often the connector should try and connect to an address from it's list.
+        /// </summary>
+        [NoTrace]
+        public virtual TimeSpan CalculateConnectionInterval()
+        {
+            return this.ConnectorPeers.Count < this.ConnectionSettings.BurstModeTargetConnections ? TimeSpans.Ms100 : TimeSpans.Second;
```

Comment:
Should we not also rename this param if burst mode is gone now?

Reply:
OK Yeah I have renamed this InitialConnectionTarget 👍